app/controllers/games_controller.rb

```diff
@@ -1,2 +1,26 @@
 class GamesController < ApplicationController
+
+	def index
+		@games = Game.all
+		@open_games = Game.where(:black_uid => nil)
```

Comment:
Yes that's the correct assumption, I just wanted to make sure it was intentional, which it clearly was.  Good work.